api/handlers.go

```diff
@@ -182,6 +233,38 @@ func (api *API) userStatsGET(w http.ResponseWriter, req *http.Request, _ httprou
 	api.WriteJSON(w, us)
 }
 
+// userPOST creates a new user.
+func (api *API) userPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	err := req.ParseForm()
+	if err != nil {
+		api.WriteError(w, err, http.StatusInternalServerError)
+		return
+	}
+	email := req.PostFormValue("email")
+	// TODO Validation
+	if email == "" {
+		api.WriteError(w, errors.New("email is required"), http.StatusBadRequest)
+		return
+	}
+	pw := req.PostFormValue("password")
+	// TODO Validation (basic complexity)
```

Comment:
I don't think we want to enforce any rules on the pw anymore.

Reply:
That was my impression as well but I wanted to confirm. I'll only ensure it's not empty.